test/hotspot/jtreg/runtime/Thread/SuspendAtExit.java

```diff
@@ -78,7 +78,30 @@ public static void main(String[] args) {
                     usage();
             }
         }
+        test(timeMax);
+
+        // Fire-up deamon that just creates new threads. This generates contention on
+        // Threads_lock while worker tries to exit, creating more places where target
+        // can be seen as handshake safe.
+        Thread threadCreator = new Thread() {
+            @Override
+            public void run() {
+                while (true) {
+                    Thread dummyThread = new Thread(() -> {});
+                    dummyThread.start();
+                    try {
+                        dummyThread.join();
+                    } catch(InterruptedException ie) {
+                    }
+                }
+            }
+        };
+        threadCreator.setDaemon(true);
+        threadCreator.start();
+        test(timeMax);
```

Comment:
Okay, but now the test is going to execute for twice `timeMax` which might be
a bit of a surprise... In particular the usage() mesg is now wrong.

Reply:
Also, the `@bug` line is not updated with this bug ID.